tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -41,59 +59,297 @@ gcloud config set compute/zone $ZONE
+
+curl -L https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/install-website.sh >install-website.sh 
+```
+
+</details>
 
 3.  Initialize a git repository in the working directory:
 
-        git init
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+git init
+```
+
+</details>
+
+## Building a Windows Image
+1.  Create new working directories using Powershell:
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```powershell
+New-Item -Name windows-image-factory -ItemType Directory
+
+Set-Location -Path ./windows-image-factory
+
+New-Item -Name scripts -ItemType Directory
+```
+
+</details>
+
+2.  Download the tutorial scripts to your local environment. To understand what's happening within the script files, scroll down to this [section](#windows-packer-scripts).
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```powershell
+# Downloading the Cloud Build config file
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/cloudbuild.yaml" -OutFile "cloudbuild.yaml"
+
+# Downloading the packer config file
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/packer.json" -OutFile "packer.json"
+
+# Downloading the powershell scripts for packer windows tutorial 
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/bootstrap-packer.ps1" -OutFile "./scripts/bootstrap-packer.ps1"
+
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/cleanup-packer.ps1" -OutFile "./scripts/cleanup-packer.ps1"
+
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/disable-uac.ps1" -OutFile "./scripts/disable-uac.ps1"
+
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/install-chocolatey.ps1" -OutFile "./scripts/install-chocolatey.ps1"
+
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/run-chocolatey.ps1" -OutFile "./scripts/run-chocolatey.ps1"
+```
+
+</details>
+
+3.  Initialize a git repository in the working directory:
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+git init
+```
+
+</details>
+
+# Task 3: Enable the required services
+Enable the Google Cloud APIs necessary for the tutorial
+
+## Building a Linux Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+gcloud services enable sourcerepo.googleapis.com
```

Comment:
combine into one command

Reply:
When you combine a lot of items into one command like this, it's a good idea to add line breaks and command-continuation characters, so that the reader doesn't end up with a long single line with a horizontal scroll bar.

Reply:
updated